pom.xml

```diff
@@ -348,6 +364,22 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.liquibase</groupId>
+                <artifactId>liquibase-maven-plugin</artifactId>
+                <version>${liquibase-maven-plugin.version}</version>
+                <configuration>
+                    <propertyFile>etc/liquibase/liquibase-maven-plugin.properties</propertyFile>
```

Comment:
Is it possible to include properties to application.properties with prefix ? 

`db.driver=net.bull.javamelody.JdbcDriver
db.url=jdbc:postgresql://localhost:5432/hyperdriver?driver=org.postgresql.Driver
db.keepAliveConnection=true
db.connectionPool=HikariCP
db.numThreads=4

liquibase.db.driver = org.postgresql.Driver
liquibase.db.url = ${db.url}
`

Reply:
Hmm, I don't think the liquibase maven plugin can deal with prefixes, unfortunately. Also, I didn't want to put the liquibase prefixes into the `application.properties` because the `LiquibaseConfiguration` directly uses `db.url` etc. so it might be misleading. E.g. you could configure `liquibase.db.url=B` and `db.url=A`, then `LiquibaseConfiguration` would just take A and ignore B.

I don't find the duplication of the db configuration that bad, because there should be no need to use the liquibase maven plugin for anything else than your local development DB

Reply:
I am fine with that. 
I didn't notice that LiquibaseConfiguration loads configuration from application.properties.